tapir/utils/management/commands/populate_functions.py

```diff
@@ -259,6 +259,7 @@ def populate_shift_templates():
     slot_name_general = ""
     slot_name_teamleader = "Teamleitung"
     slot_name_cleaning = "Reinigung & Aufräumen"
+    slot_name_welcomesession = "Welcome Session"
```

Comment:
This is the code that generates the test data (when running `python manage.py populate --reset_all`), since it doesn't generate any welcome session shift, we don't need this line, nor L316-317 I would say?

Reply:
Morning Theo. I used it for Testing and left it in there if others want to use it, but can remove it ofc.